.github/helm-repo.sh

```diff
@@ -2,8 +2,6 @@
 set -euxo pipefail
 
 helm repo add hazelcast "https://hazelcast-charts.s3.amazonaws.com/"
-helm repo add influx "https://helm.influxdata.com/"
 helm repo add bitnami "https://charts.bitnami.com/bitnami"
 helm repo add bitnami-full "https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami"
```

Comment:
not sure what affects these two files, but if all references of bitnami have been removed, why these lines are still present?
ct-lint.yaml: line 6-7, `bitnami=...` , `bitnami-full=...`
helm-repo.sh: line 5-6, `helm repo add bitnami...`, `helm repo add bitnami-full...`